### PR TITLE
pacman-s: add alias page; pacman-sync: update page

### LINKED ISSUE #14003: Let's document: pacman options in shortform
### Command description

Almost everywhere you look, pacman documentation and guides use shortform for its options, including the manpage. The main pacman.md page uses these the way you see everywhere else, but the subcommands use longform. I suggest that we convert these pages to look more like what people are used to and use mnemonics and the page title to indicate what it means.

### Documentation

https://manned.org/pacman.8

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### Commands

- [x] [pacman](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman.md)
- [x] [pacman-database](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-database.md)  #15177
- [x] [pacman-query](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-query.md)  #15180
- [x] [pacman-remove](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-remove.md) #14807
- [x] [pacman-sync](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-sync.md) #14658
- [x] [pacman-deptest](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-deptest.md) #15178
- [x] [pacman-upgrade](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-upgrade.md) #15179
- [x] [pacman-files](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-files.md) #14858

### PR DESCRIPTION
**Pull request by angadsgrover**
#14003

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

**Comment by angadsgrover** [comment]
@Managor and @cyqsimon  
Please advise if the filename and content aligns with your expectations for #14003 . If so, I can create similar updates for the other Pacman options and submit them as separate pull requests for better organization and review.
Also please advise if you would prefer replacing the long form pages or creating new ones with similar naming scheme as to this one

**Comment by cyqsimon** [comment]
I'm a bit torn on this as well. We currently don't seem to have a convention for commands whose primary verb is a flag, as is the case with `pacman`. Maybe we should put this PR on hold and sort that out first.

**Comment by angadsgrover** [comment]
@sebastiaanspeck 
I'm unable to name the file pacman -S since the naming convention restricts names to lowercase. I have since named the file pacman-s

@cyqsimon 
I understand, would you want me to delete this PR then? I'll focus on the other issues and some other missing commands I wish to add to tldr.

**Comment by cyqsimon** [comment]
> I understand, would you want me to delete this PR then?

No, no need for that. I was just pointing out that we probably should standardise this sort of thing.

**Comment by Managor** [comment]
We already have pacman-sync.md. This page should be made into an alias.

**Comment by Managor** [comment]
The way I see it is that the best way to handle this is to edit the commands in [pacman-sync.md](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-sync.md) to match what people use. Leave the filename and title of that page as is. [pacman.md](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman.md) already has a suggestion to check out the pacman-sync page. Edit this page to be an [alias](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md) of that page.

**Comment by angadsgrover** [comment]
Just to clarify, 
We should edit the shortform commands into the main [pacman-sync](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-sync.md) page and add an alias for the pacman-s page with the full form commands in the pacman-s page?

**Comment by Managor** [comment]
What do you mean with "full form commands in the pacman-s page"?

**Comment by angadsgrover** [comment]
By full form commands, I mean the `--sync --refresh --upgrade` type commands which are currently in the [pacman-sync.md](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-sync.md) page

**Comment by Managor** [comment]
No, just make this page into an alias. See other pages like https://github.com/tldr-pages/tldr/blob/main/pages/common/trash-cli.md for example. If you want, you can use [mnemonics](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#short-option-mnemonics) to clarify

File: pages/linux/pacman-s.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+`pacman -Si {{package}}`
+
+- Synchronize and update all packages, but ignore a specific package (can be used more than once):
+
+`sudo pacman -Syu --ignore {{package}}`
+
+- Remove not installed packages and unused repositories from the [c]ache (use `-Scc` flag to clean all packages):
+
+`sudo pacman -Sc`
```
[the remark this answers is not in the document]
Comment (angadsgrover) [reply pages/linux/pacman-s.md:38]: I understand this page being an alias, but without including the short form commands, how do we solve the initial issue. Should we do mnemonic in the original [pacman-sync.md](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman-sync.md) then?

[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/pacman-s.md:38]: Yes, that is what I asked.

**Comment by Managor** [comment]
Now all that is left is converting the commands to shortform

File: pages/linux/pacman-s.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# pacman-s
+
+> This command is an alias of `pacman-sync`.
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/pacman-s.md:3]: Are you sure this is ok? This is about pointing to another page and not the command itself.

File: pages/linux/pacman-sync.md [hunk 3]
```diff
@@ -6,32 +6,32 @@
...
+`sudo pacman -S {{package}}`
 
-- Synchronize and update all packages (add `--downloadonly` to download the packages and not update them):
+- [S]ynchronize and [u]pdate all packages along with a sys[u]pgrade (add `--downloadonly` to download the packages and not update them):
 
-`sudo pacman --sync --refresh --sysupgrade`
+`sudo pacman -Syu`
 
-- Update all packages and install a new one without prompting:
+- [U]pdate all packages and install a new one without prompting:
```
[the remark this answers is not in the document]
Comment (angadsgrover) [reply pages/linux/pacman-sync.md]: Updated it to Update and [u]pgrade

File: pages/linux/pacman-s.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# pacman-s
```
Comment (Managor) [inline pages/linux/pacman-s.md:1]:
```suggestion
# pacman -S
```
@sebastiaanspeck What do you think, should the title reflect the actual command and differ from the filename in this case?

File: pages/linux/pacman-sync.md [hunk 5]
```diff
@@ -6,32 +6,32 @@
...
+`sudo pacman -Syu --overwrite {{path/to/file}}`
 
-- Synchronize and update all packages, but ignore a specific package (can be used more than once):
+- [S]ynchronize and [u]pdate all packages, but ignore a specific package (can be used more than once):
 
-`sudo pacman --sync --refresh --sysupgrade --ignore {{package}}`
+`sudo pacman -Syu --ignore {{package}}`
 
-- Remove not installed packages and unused repositories from the cache (use two `--clean` flags to clean all packages):
+- Remove not installed packages and unused repositories from the cache (use `Scc` flag to [c]lean all packages):
```
Comment (Managor) [inline pages/linux/pacman-sync.md:35]:
```suggestion
- Remove not installed packages and unused repositories from the cache (use the flags `Scc` to [c]lean all packages):
```
Each shortform letter is its own flag so the language should reflect it.

**Review by Managor — APPROVED** [review]
Looks good to me once the open issues have been resolved. Thanks for taking your time to go through with this.

Remember that none of this is set in stone. If you feel like the format of this page could be improved for clarity, just fire another PR.

**Comment by angadsgrover** [comment]
Thank you for the corrections and feedback for the PR.
I'll try and implement similarly for the other pacman commands once this PR is completed.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**